packages/share_plus/share_plus_platform_interface/lib/method_channel/method_channel_share.dart

```diff
@@ -69,6 +70,26 @@ class MethodChannelShare extends SharePlatform {
     return channel.invokeMethod('shareFiles', params);
   }
 
+  /// Summons the platform's share sheet to share multiple files.
+  @override
+  Future<void> shareXFiles(
+    List<XFile> files, {
+    String? subject,
+    String? text,
+    Rect? sharePositionOrigin,
+  }) {
```

Comment:
Is it necessary to have the `shareXFiles` method in the platform interface, rather than being a private method of the web implementation?

Reply:
This is also solved because of the deprecations